pkg/ansible/controller/reconcile.go

```diff
@@ -186,7 +186,7 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 				return reconcile.Result{}, err
```

Comment:
It is missing the fragment with the bug fix which will be used to gen the CHANGELOG.
if it was not added in master we also need a PR with it there as well.

Reply:
Oh ok. I'll make that change here & open a PR on master as well then. Thanks.